rialto_airflow/dags/harvest.py

```diff
@@ -150,6 +159,9 @@ def publish(pubs_to_contribs, merge_publications):
 
     openalex_jsonl = openalex_harvest(snapshot)
 
```

Comment:
Eventually we'll want to tie the WoS harvesting into the workflow, but for now nothing is dependent on it.